pkg/test/resourcefixture/testdata/basic/networksecurity/v1beta1/networksecurityauthorizationpolicy/networksecurityauthorizationpolicy-full/_http.log

```diff
@@ -105,65 +108,122 @@ X-Frame-Options: SAMEORIGIN
 X-Xss-Protection: 0
 
 {
-  "done": true,
```

Comment:
This looks problematic - are we not waiting for the operation to complete?

Reply:
fixed

Reply:
(Or rather not change vs the DCL logic)